contrib/roles/windows/kubernetes/tasks/run_minion_init.yml

```diff
@@ -27,9 +57,8 @@
     line: |
       $RegROPath = "HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Run"
       Remove-ItemProperty $RegROPath "OVN_Init_Node" -ErrorAction SilentlyContinue
-      # TODO: Remove the sleep for the next release of OVS
-      sleep 30
-      {{ install_info.install_path }}\\ovnkube.exe --k8s-kubeconfig={{ install_info.install_path }}\\kubeconfig.yaml --k8s-apiserver http://{{ kubernetes_info.MASTER_IP }}:8080 --init-node {{ ansible_hostname|lower }} --k8s-token {{TOKEN}} --nb-address "tcp://{{ kubernetes_info.MASTER_IP }}:6641" --sb-address "tcp://{{ kubernetes_info.MASTER_IP }}:6642" --cluster-subnet {{ kubernetes_info.CLUSTER_SUBNET }} --cni-conf-dir="{{ install_info.install_path }}/cni" --cni-plugin "ovn-k8s-cni-overlay.exe"
+      Start-Service ovn-kubernetes-node
+      sleep 15
       ovs-vsctl set open_vswitch . external_ids:ovn-encap-ip="{{ host_public_ip }}"
```

Comment:
The sleep and setting ovn-encap-ip can be removed if ``--encap-ip={{ host_internal_ip }}`` is passed to ovnkube.

Reply:
Same as above. I will add it as a workaround and open an issue for the IP detection.